Apps/Sandcastle/gallery/MSAA.html

```diff
@@ -79,6 +79,26 @@
           viewer.scene.camera.lookAt(target, offset);
         }
 
+        let currentAssetId;
+        async function createTileset(assetId) {
```

Comment:
Do we also need to write an async version of the `createModel` function above? (line 48, 67... sorry but https://github.com/github/roadmap/issues/347)

Reply:
Nope, model entities handle the async model primitive creation through modelVisualizer, and the error is logged. This is similar for other visualizers which already exist.